corgi/tasks/tagging.py

```diff
@@ -0,0 +1,58 @@
+from celery.utils.log import get_task_logger
+from celery_singleton import Singleton
+
+from config.celery import app
+from corgi.core.models import Product, ProductStream, ProductStreamTag, ProductVersion
+from corgi.tasks.common import RETRY_KWARGS, RETRYABLE_ERRORS
+
+logger = get_task_logger(__name__)
+
+
+@app.task(
+    base=Singleton,
+    autoretry_for=RETRYABLE_ERRORS,
+    retry_kwargs=RETRY_KWARGS,
+)
+def apply_tags():
+    apply_stream_no_manifest_tags()
+
+
+def apply_stream_no_manifest_tags():
```

Comment:
I think we can just combine the two methods since the first is just a one-liner that calls the second:
```suggestion
def apply_stream_no_manifest_tags() -> None:
```
Should we also schedule this as a daily task, so that any new RHEL / middleware / managed services streams are automatically updated with this tag?

Reply:
Ok, I set this up this way as I suspect we will have other tagging tasks to perform in future, but we can always refactor it back out when/if adding those. I also added  a daily task for this job, thank you!